tests/testthat/test-trailing_whitespace_linter.R

```diff
@@ -43,3 +43,30 @@ test_that("also handles completely empty lines per allow_empty_lines argument",
     trailing_whitespace_linter(allow_empty_lines = TRUE)
   )
 })
+
```

Comment:
fix the `implicit_integer_linter` hits while we're here in this file? (see above in UI)